ci-operator/config/open-cluster-management/api/.config.prowgen

```diff
@@ -1,2 +1 @@
-private: true
```

Comment:
We can delete the files entirely. `expose: true` is only effective with `private: true`